frontend/src/pages/modelServing/screens/projects/DeleteServingRuntimeModal.tsx

```diff
@@ -68,8 +68,12 @@ const DeleteServingRuntimeModal: React.FC<DeleteServingRuntimeModalProps> = ({
               onBeforeClose(true);
             })
             .catch((e) => {
-              setError(e);
-              setIsDeleting(false);
+              if (e.response.status === 404) {
+                onBeforeClose(true);
+              } else {
+                setError(e);
+                setIsDeleting(false);
+              }
```

Comment:
I don't think this code is ever executed.

Reply:
If allSettled is not catching then is right, I'll delete it